buildSrc/src/main/java/org/elasticsearch/gradle/test/rest/CopyRestApiTask.java

```diff
@@ -132,7 +132,11 @@ void copy() {
             project.copy(c -> {
                 c.from(project.zipTree(coreConfig.getSingleFile()));
                 c.into(getTestSourceSet().getOutput().getResourcesDir()); // this ends up as the same dir as outputDir
-                c.include(includeCore.get().stream().map(prefix -> COPY_TO + "/" + prefix + "*/**").collect(Collectors.toList()));
+                if (includeCore.get().isEmpty()) {
+                    c.include(COPY_TO + "/**");
```

Comment:
The fact we are calling the path prefix for api specs `COPY_TO` I think makes this hard to follow. I'm thinking "why are we referencing the copy _destination_ when filtering _sources_?". If we are going to use this constant for both purposes, we should rename it appropriately, something like `API_SPEC_PREFIX`.

Reply:
Renamed `COPY_TO` to `REST_API_PREFIX` and `REST_TEST_PREFIX` on  https://github.com/elastic/elasticsearch/pull/53184/commits/f8459c9420d05ec90b70bbc75401ea5e1d84254e